nipype/interfaces/io.py

```diff
@@ -2863,3 +2864,49 @@ def _list_outputs(self):
 
     def _add_output_traits(self, base):
         return add_traits(base, list(self.inputs.output_query.keys()))
+
+
+class ExportFileInputSpec(BaseInterfaceInputSpec):
+    in_file = File(exists=True, mandatory=True, desc='Input file name')
+    out_file = File(mandatory=True, desc='Output file name')
+    check_extension = traits.Bool(True, desc='Ensure that the input and output file extensions match')
+    clobber = traits.Bool(desc='Permit overwriting existing files')
+
+
+class ExportFileOutputSpec(TraitedSpec):
+    out_file = File(exists=True, desc='Output file name')
+
+
+class ExportFile(SimpleInterface):
+    """ Export a file to an absolute path
+
+    This interface copies an input file to a named output file.
+    This is useful to save individual files to a specific location,
+    instead of more flexible interfaces like DataSink.
+
+    Examples
+    --------
+
+    >>> from nipype.interfaces.io import ExportFile
+    >>> import os.path as op
+    >>> ef = ExportFile()
+    >>> ef.inputs.in_file = "T1.nii.gz"
+    >>> os.mkdir("output_folder")
+    >>> ef.inputs.out_file = op.abspath("output_folder/sub1_out.nii.gz")
+    >>> ef.run()
```

Comment:
This returns an `InterfaceResult` object.

```suggestion
    >>> res = ef.run()
```

And maybe you can follow it up with an:

```Python
>>> os.path.exists(res.outputs.out_file)
True
```